content/ko/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -80,7 +80,7 @@ Cannot determine if job needs to be started. Too many missed start time (> 100).
 
 ## {{% heading "whatsnext" %}}
 
-[크론 표현 포맷](https://pkg.go.dev/github.com/robfig/cron?tab=doc#hdr-CRON_Expression_Format)은
+[크론 표현 포맷](https://en.wikipedia.org/wiki/Cron)은
```

Comment:
I recommend hyperlinking to https://ko.wikipedia.org/wiki/Cron because https://ko.wikipedia.org/wiki/Cron is in Korean.

@kubernetes/sig-docs-ko-reviews thoughts?

PS @morty it's usual to put in fixes into the English localization and then let other localization teams pick up fixes.

Reply:
@sftim I agree with you.

Reply:
OK, I have updated the link to point to the Korean Wikipedia page now.